src/commands/slash/suggest.ts

```diff
@@ -21,7 +20,9 @@ export const command = new SlashCommand(
 
 			if (suggestionChannel.isThread() || !suggestionChannel.isText()) return;
 
-			const message = suggestionChannel.messages.cache.get("931837510408089622") ?? await suggestionChannel.messages.fetch("931837510408089622");
+			const message =
+				suggestionChannel.messages.cache.get("931837510408089622") ??
+				(await suggestionChannel.messages.fetch("931837510408089622"));
```

Comment:
This is the worst thing I've ever seen
```suggestion
			const message = await suggestionChannel.messages.fetch("931837510408089622");
```
Discord.js already checks if the message is cached and returns it in the case